base/container_scripts/db_restore.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+declare FILENAME="$1"
+
+set -e
```

Comment:
I think it should fail if $1 was not set.
```suggestion
set -eu

declare FILENAME="$1"
```